docs/variants/novacustom_ns5x_tgl/hardware-matrix.md

```diff
@@ -5,6 +5,16 @@
 This document describes the hardware configurations used for validation of the
 coreboot port on the NovaCustom NS51/NS70 laptops.
 
+## Ports specification
+
```

Comment:
@Pokisiekk Instead of adding explanations on the graphic, couldn't we gather them in a table (e.g., with an additional column)?

The proposed approach is problematic in that the graphics will have to be updated with each change.

Reply:
Done